<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALineMarkerFillDialog</class>
 <widget class="QDialog" name="ALineMarkerFillDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QPushButton" name="pbUpdateObject">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>UpdateObject</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frOptions">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>95</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Root draw options:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leOptions">
        <property name="toolTip">
         <string>&quot;same&quot; will be automatically added to options for all except the first drawing object</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbOptionsHelp">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>https://root.cern/doc/master/classTHistPainter.html</string>
        </property>
        <property name="text">
         <string>help</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>95</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Line</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frLine">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cobLineStyle"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbLineWidth"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbLineColor">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frLineCol">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>23</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labInvis1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>73</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>73</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Marker</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frMarker">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cobMarkerStyle"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ledMarkerSize">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbMarkerColor">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frMarkerCol">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>23</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbMarkerColorAsLine">
        <property name="text">
         <string> as line</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Fill</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frFill">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cobFillStyle">
        <item>
         <property name="text">
          <string>Hollow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pattern</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frFillPattern">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>#</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbFillPattern">
           <property name="minimum">
            <number>3001</number>
           </property>
           <property name="maximum">
            <number>3999</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbFillColor">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frFillCol">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>23</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbFillColorAsLine">
        <property name="text">
         <string> as line</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbPreview">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sbLineWidth</sender>
   <signal>editingFinished()</signal>
   <receiver>pbUpdateObject</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cobLineStyle</sender>
   <signal>activated(int)</signal>
   <receiver>pbUpdateObject</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cobMarkerStyle</sender>
   <signal>activated(int)</signal>
   <receiver>pbUpdateObject</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledMarkerSize</sender>
   <signal>editingFinished()</signal>
   <receiver>pbUpdateObject</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cobFillStyle</sender>
   <signal>activated(int)</signal>
   <receiver>pbUpdateObject</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbFillPattern</sender>
   <signal>editingFinished()</signal>
   <receiver>pbUpdateObject</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leOptions</sender>
   <signal>editingFinished()</signal>
   <receiver>pbUpdateObject</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
